integration/test_contention_state_transitions.py

```
@@ -0,0 +1,718 @@
+        self._create_text_index(client)
+        client.execute_command("HSET", "doc:1", "content", "hello world")
+        client.execute_command("HSET", "doc:2", "content", "hello there")
+        IndexingTestHelper.is_indexing_complete_on_node(client, "idx")
+
+        client.execute_command("FT._DEBUG", "PAUSEPOINT", "SET", "mutation_processing")
+        hset1_t, _, _ = self._hset_blocking("doc:1", "updated1")
+        hset2_t, _, _ = self._hset_blocking("doc:2", "updated2")
+        # Two writer threads will both park at mutation_processing.
+        _wait_paused(client, "mutation_processing", expected=1)
+
+        search_t, search_res, search_err = self._search_blocking()
```

Ruff (0.15.12)

[warning] Unpacked variable `search_res` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)